runtime/oti/j9nonbuilder.h

```diff
@@ -5647,6 +5655,10 @@ typedef struct J9VMThread {
 #endif /* defined(J9VM_OPT_JFR) */
 } J9VMThread;
 
+#if defined(J9VM_ENV_DATA64)
+#define J9VMTHREAD_ARRAYLET_ENABLED(vmThread) (J9IndexableObjectLayout_ArrayletMask & (vmThread)->indexableObjectLayout)
+#define J9VMTHREAD_IS_INDEXBLEDATAADDRPRESENT(vmThread) (J9IndexableObjectLayout_DataAddrMask & (vmThread)->indexableObjectLayout)
```

Comment:
add extra '_' for easier reading J9VMTHREAD_IS_INDEXBLE_DATAADDR_PRESENT
add 'IS' to  J9VMTHREAD_IS_ARRAYLET_ENABLED